numpy/_core/defchararray.pyi

```diff
@@ -609,33 +609,33 @@ def capitalize(a: _StringDTypeSupportsArray) -> _StringDTypeArray: ...
 def capitalize(a: T_co) -> _StringDTypeOrUnicodeArray: ...
 
 @overload
-def center(a: U_co, width: i_co, fillchar: U_co = ...) -> NDArray[str_]: ...
+def center(a: U_co, width: i_co, fillchar: U_co = " ") -> NDArray[str_]: ...
 @overload
-def center(a: S_co, width: i_co, fillchar: S_co = ...) -> NDArray[bytes_]: ...
+def center(a: S_co, width: i_co, fillchar: S_co = " ") -> NDArray[bytes_]: ...
```

Comment:
I suppose `fillchar` is wrong then, and should instead be something that accepts `str`, e.g.:

```suggestion
def center(a: S_co, width: i_co, fillchar: str | S_co = " ") -> NDArray[bytes_]: ...
```